### ISSUE TITLE
Discrete Graphics Card Usage in Program

### ISSUE THREAD

**Issue by SPluzh** [emoacht/Monitorian#447]
In my laptop the program currently utilizes a discrete graphics card. Can it work with the integrated graphics card?
<img width="438" alt="NVIDIA_Share_qSSkJyUL9T" src="https://user-images.githubusercontent.com/131027290/232572907-25f524cb-d4ae-4d2e-82d5-29edc0bda5d5.png">

I suppose this can be done as there is a similar software (https://github.com/xanderfrangos/twinkle-tray).

**Comment by emoacht** [comment]
Could you elaborate the purpose and post probe.log?

**Comment by SPluzh** [comment]
I have a laptop with discrete and integrated graphics cards. Usually programs use the built-in video card. But in this case Monitorian uses the discrete video card. The discrete graphics card goes to sleep most of the time for power efficiency. Usually the discrete graphics card is used by games, 3D applications and other highly loaded programs.

**Comment by emoacht** [comment]
Thanks for the explanation.
I think it is not determined by each app but up to the setting of the OS.
Then, have you tried to set the GPU to be used in the OS"s settings?
https://pureinfotech.com/set-default-gpu-apps-windows-11/

**Comment by SPluzh** [comment]
I turned this setting on, but it didn't help.
<img width="747" alt="ApplicationFrameHost_ASPPGXDipn" src="https://user-images.githubusercontent.com/131027290/232724297-516e0dc3-b283-4abb-bfb9-a48731c77a80.png">
But I found out that Monitorian uses a discrete graphics card when an external monitor is connected.
If you disconnect the external monitor, Monitorian will use the built-in video card. 
Other programs use the built in graphics card regardless of the external monitor connection.
